docs/lang/articles/get-started/index.md

```diff
@@ -268,28 +270,6 @@ Taichi, as it will only loop over active elements in a sparse field. In
 dense fields, all elements are active.
 :::
 
-:::caution
-
-Struct-for loops must live at the outer-most scope of kernels.
-
-It is the loop **at the outermost scope** that gets parallelized, not
-the outermost loop.
-
-```python
-@ti.kernel
-def foo():
-    for i in x:
-        ...
-
-@ti.kernel
-def bar(k: ti.i32):
-    # The outermost scope is a `if` statement
-    if k > 42:
-        for i in x: # Not allowed. Struct-fors must live in the outermost scope.
-            ...
-```
```

Comment:
This part is for struct-for loops. It's similar to range-fors, but they are different. So we should not remove it (maybe we should highlight the differences to prevent people from thinking they are the same).